doc/man3/SSL_CTX_set_session_ticket_cb.pod

```diff
@@ -66,67 +66,110 @@ application that a session ticket has just been decrypted.
 =head1 NOTES
 
 When the B<dec_cb> callback is invoked, the SSL_SESSION B<ss> has not yet been
-assigned to the SSL B<s>. The B<retv> indicates the result of the ticket
-decryption which can be modified by the callback before being returned. The
-callback must check the B<retv> value before performing any action, as it's
-called even if ticket decryption fails.
+assigned to the SSL B<s>. The B<status> indicates the result of the ticket
+decryption. The callback must check the B<status> value before performing any
+action, as it is called even if ticket decryption fails.
```

Comment:
I don't think so. It would be quite unexpected if the callback were invoked after we'd already encountered a fatal error. I don't think we explicitly need to say it.

Reply:
Actually, it is still true, it is called when no matching is found, or the HMAC fails. That's the NO_DECRYPT case.

Reply:
> I don't think so. It would be quite unexpected if the callback were invoked after we'd already encountered a fatal error. I don't think we explicitly need to say it.

Fair enough. Just want to make sure it's clear.